src/content/docs/en/guides/upgrade-to/v6.mdx

```diff
@@ -371,6 +371,37 @@ prefetch('/about');
 
 <ReadMore>Learn more about [prefetching](/en/guides/prefetch/).</ReadMore>
 
+### Experimental Flags
```

Comment:
@florian-lefebvre I added this section (copied from v5 guide) so that we could show removing experimental flags, too. Then below, we can mention this section as where to go to see how to remove (all) flags, rather than add more to the listing itself.